packages/vite/src/node/plugins/asset.ts

```diff
@@ -250,7 +244,7 @@ export function assetPlugin(config: ResolvedConfig): Plugin {
       if (
         config.command === 'build' &&
         config.build.ssr &&
-        !config.build.ssrEmitAssets
+        !config.build.emitAssets
```

Comment:
Is `ssrEmitAssets` completely ignored in Vite 6?
I started to test Vite 6 on my vite-plugins repo https://github.com/hi-ogawa/vite-plugins/pull/297 and it looks like `ssrEmitAssets` is no-op and requires switching to `emiAssets`.

Should `ssrEmitAssets` be still functional for old ssr build as v5 compat?

Reply:
Yes. Thanks for the PR!